rpmostree-cxxrs.cxx

```diff
@@ -6094,16 +6124,6 @@ cache_branch_to_nevra (::rust::Str nevra) noexcept
   rpmostreecxx$cxxbridge1$cache_branch_to_nevra (nevra, &return$.value);
   return ::std::move (return$.value);
 }
-
```

Comment:
Any reason this code is moved? Maybe split this change into another commit?

Reply:
It was a result of me running `./ci/verify-cxx.sh`, I can revert if you'd like though.

Reply:
OK, then maybe let's split this change into its own commit?

Reply:
This is minor so optional.